lib/capability/capability_test.go

```diff
@@ -236,10 +243,33 @@ func TestSetCapabilitiesWithImplicitlyEnabled(t *testing.T) {
 			priorEnabled: map[configv1.ClusterVersionCapability]struct{}{"cap1": {}, "cap2": {}, "cap3": {}},
 			wantImplicit: []string{"cap1", "cap2", "cap3"},
 		},
+		{name: "set capabilities 4_11 with always enabled",
```

Comment:
nit, this feels more informative:
```suggestion
		{name: "set baseline capabilities with always enabled",
```

Reply:
I copied the description from the existing test. But I think you are right, `baseline` would be a neutral term. Also, I modified the baseline parameter to be the current set to avoid mentioning the EOL release (4.11).